src/Umbraco.Web.UI.Client/src/common/directives/components/forms/umbradiobutton.directive.js

```diff
@@ -49,7 +49,7 @@
 
         function onInit() {
             vm.inputId = vm.inputId || "umb-radio_" + String.CreateGuid();
-
+            vm.disableDirtyCheck = vm.disableDirtyCheck === true;
```

Comment:
Same as above: Is this necessary? And won't it break backwards compat with people using a truthly value that's not strictly `true`?

Reply:
Not sure if we need to support both `true/false` and `'true'`/`'false'` as string values. Here is just do the comparison in the controller so the view is a bit cleaner :)

Reply:
While certainly cleaner it's also more stringent. My point is that the directive (presumably) supports truthly values today, so it should keep on doing that to preserve compatibility with current usages in 3rd party extensions etc.

Reply:
We can add more checks here to make it work with other truthly values. E.g.

```
vm.disableDirtyCheck = vm.disableDirtyCheck === true || vm.disableDirtyCheck === 'true' || vm.disableDirtyCheck === 1;
```

Reply:
I noticed we have this `isBool` function in color picker:
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/views/propertyeditors/colorpicker/colorpicker.controller.js#L142-L145

Maybe we should have a global function somewhere to check for truthly values?

Reply:
Come to think of it we already do - there's an `Object.toBoolean(value)` method (defined in Extensions.js).